.gitignore

```diff
@@ -117,4 +117,6 @@ dist
 
 .expo
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+package-lock.json
```

Comment:
At the same time we probably don't want the application itself to ignore package-lock— I think we just want to let them generate it for themselves.

Reply:
shrexpo's own .gitignore isn't consumed on installation, so this _should_ be chill. If there's a better way though lmk!

Reply:
Ah got it, excellent 👍 I think that is ideal.